.travis.yml

```diff
@@ -1,22 +1,12 @@
 language: android
+
 android:
   components:
-    - platform-tools
-    - tools
-
-    # The BuildTools version used by your project
-    - build-tools-22.0.1
-
-    # The SDK version used to compile your project
-    - android-22
-
-    # Additional components
-    # - extra-google-google_play_services
-    # - extra-google-m2repository
+    - build-tools-23.0.1
+    - android-23
+    - extra-android-support
```

Comment:
You are using android 22 instead of android 23. and also what I said in the PR 1º comment:

"The problem you had with your Travis-CI configuration it's not related to the application build. The problem is in your artefacts repository hosted at http://ci.novoda.com/maven/releases which is completely down or not accesible from the Travis-CI environment. Http requests to this repository were getting timeouts continuously and that's why all your Travis-CI builds were failing."